src/profile/main.mo

```diff
@@ -6,28 +6,91 @@
  * Stability  : Experimental
  */
 
+import Array "mo:stdlib/array.mo";
 import Blob "mo:stdlib/blob.mo";
+import Nat "mo:stdlib/nat.mo";
+import Option "mo:stdlib/option.mo";
+
+import Graph "canister:graph";
 
 import Directory "./directory.mo";
 import Types "./types.mo";
 
+type NewProfile = Types.NewProfile;
 type Profile = Types.Profile;
 type PrincipalId = Types.PrincipalId;
 
 actor Profile {
   var directory : Directory.Directory = Directory.Directory();
+  directory.seed();
+
+  public func healthcheck () : async Bool {
+    let isGraphAlive : Bool = await Graph.healthcheck();
+    true and isGraphAlive
```

Comment:
Why the odd `let` and `and` instead of just a one-liner `await Graph.healthcheck();`?